lib/glimesh/channel_lookups.ex

```diff
@@ -119,19 +127,21 @@ defmodule Glimesh.ChannelLookups do
 
   def get_channel_by_hmac_key(hmac_key) do
     Repo.one(
-      from c in Channel,
+      from(c in Channel,
         where: c.hmac_key == ^hmac_key and c.inaccessible == false
+      )
     )
     |> Repo.preload([:category, :user])
   end
 
-  def get_channel_for_user(user) do
+  def get_channel_for_user(user, include_inaccessible \\ false) do
     Repo.one(
-      from c in Channel,
+      from(c in Channel,
         join: u in User,
         on: c.user_id == u.id,
         where: u.id == ^user.id,
-        where: c.inaccessible == false
+        where: c.inaccessible == ^include_inaccessible
```

Comment:
So when `include_inaccessible` is set to true it will **only** find rows where `inaccessible` is true. I ran into this issue when including banned users in searches. I couldn't find an easy way to conditionally adding a new where clause.

Reply:
Just made a new function to not introduce confusion.